src/domains/my/detail/MyPost.jsx

```
@@ -1,22 +1,61 @@
+    freeError,
+    jobError,
+    fetchFreeBoard,
+    fetchJobBoard,
+  } = useMyBoardStore()
+
+  useEffect(() => {
+    fetchFreeBoard()
+    fetchJobBoard()
+  }, [fetchFreeBoard, fetchJobBoard])
+
+  const currentList = choiceBoard === '자유게시판' ? freeBoard : jobBoard
```

ESLint

[error] 'currentList' is assigned a value but never used.

(no-unused-vars)